data-structures-linked-lists/includes.js

```diff
@@ -0,0 +1,11 @@
+/* exported includes */
+
+function includes(list, value) {
+  let copy = list;
```

Comment:
Careful, this is not a copy. Fortunately we don't need a copy, we are _totally_ supposed to mutate this data structure.